br/pkg/utils/backoff.go

```diff
@@ -86,6 +86,11 @@ func (rs *RetryState) RecordRetry() {
 	rs.retryTimes++
 }
 
+// ReduceRetry reduces retry times for 1.
+func (rs *RetryState) ReduceRetry() {
+	rs.retryTimes--
```

Comment:
should we check if `retryTimes > 0`?

Reply:
retryTimes can be -1 for a short time and soon it will be increased to 0. I think we can tolerate it.